CHANGES.md

```diff
@@ -10,6 +10,9 @@ To be released.
 
 ### Backward-incompatible API changes
 
+ -  `BlockCompletion<TPeer, TAction>.Complete()` no longer accepts
+    an argument to regulate a single session.  [[#1961]]
```

Comment:
It would be better if it mentions the removed parameter names.

Reply:
See ad0012a19e9530016f1f45de0baee110fec65f3d.